rotkehlchen/data/all_assets.json

```diff
@@ -5420,11 +5429,13 @@
         "type": "ethereum token"
     },
     "GNT": {
+        "active": false,
```

Comment:
It's still traded. We don't use the active attribute, anywhere yet, but don't show it as inactive if it's not completely dead.